content/docs/quickstart/symphony.md

```diff
@@ -1,14 +1,16 @@
 ---
-title: Run a Symphony app
+title: Run a Symfony app
 ---
 
-Symfony is a framework for building web applications in PHP. Symfony uses Doctrine library to access database. Using Neon from Symfony + Doctrine is straightforward and differs nothing from using a vanilla postgresql.
+### Introduction
 
-First, obtain secret token from “Connection details” panel:
+Symfony is a framework for building web applications in PHP. Symfony uses Doctrine library to access database. Using Neon from Symfony + Doctrine is straightforward and differs nothing from using a vanilla PostgreSQL.
 
-1. Select Project you wish to use in the UI. Click on it.
-2. Click “Generate token” link.
-3. Substitue token placeholder below with this token.
+## Step 1 — Set up a Neon Project
+
+## Step 2 — Select Project in the UI
```

Comment:
Nit: this is two steps here, but a single step somewhere in previous files.

Reply:
@yeputons 

In the original [Symphony doc](
https://neon.tech/docs/quickstart/symphony/), the steps are listed like so: 

```
First, obtain secret token from “Connection details” panel:

    Select Project you wish to use in the UI. Click on it.
    Click “Generate token” link.
    Substitue token placeholder below with this token
```
There are no steps telling the user to set up a Neon Project or how to do so, or to use an existing one if already created. It assumes the user already has a project.

In the recently updated Hasura doc (in this PR), the steps are listed like so,

```
## Step 1 — Set up a Neon Project

In this step, you will create or select an existing Neon Project to be used as a data source for Hasura.

First, go to the [Neon console](https://console.neon.tech).

Next, you will obtain the connection details. Keep your connection details as we will need them later.

Then, open the Project page, and copy the connection string (`DATABASE_URL`) if you created a new Project.

Note: The following method bellow will disconnect all existing clients

Otherwise, if you are using a previously created Project, there are two options: either create a new user or reset password of an existing user to get the complete connection string.
```

To create some consistency amongst the quickstart-docs, I attempted to implement a standard across documentation.

We can combine the steps in Symfony doc as one step, ex. Step 1 - Set up a Neon Project and Select Project in the UI but these are two actions a user must take in one step.

Reply:
Hi @yeputons just following up on Alexandria's message above. Please let us know your thoughts.

Reply:
@alexandriastech @Harleta 

I'm all for consistency in the first place. I think my comment here does not matter much here at the moment because there are no instructions written in these steps currently, so it does not make much sense to compare yet-to-be-written instructions with others for consistency.

My understanding is that in all cases (except for a direct postgres connection via `psql`) a user has to: a) either create a Neon project or go to the dashboard of an existing one; b) copy some subset of connection credentials; c) put them into a quickstart-specific place in a specific format. In case of `psql` we provide a wizard-like hint in our server's output if the user don't have proper credentials.